### DISCUSSION TITLE
Difference between ATEPC models

### DISCUSSION THREAD

**Discussion by kerolzeeq** [yangheng95/PyABSA#221] [Q&A]
Hello, I'm a bit curious and I have 2 questions on the ATEPC models.

1. What's the difference and similarities between LCF-ATEPC model and FAST-LCF-ATEPC model?

2. What's the difference and similarities between the normal LCF-ATEPC model and LCFS-ATEPC model?

**Comment by yangheng95** [comment]
They basicly share the samilar architectures. 
The  fast* model use one bert for global and local context, and only forward once.
The *s models use syntax-based SRD
See ref: https://www.sciencedirect.com/science/article/pii/S0925231220312534
https://aclanthology.org/2020.acl-main.293/

**Reply by kerolzeeq** [reply]
By *'use one bert for global and local context'*. 
Do you mean there is only one bert for both global and local context OR 
there are 2 berts, one for global context and the other for local context?

**Reply by yangheng95** [reply]
There is one bert for both global context and the other for local context. If there are two berts, it is the standard LCF-ATEPC

**Reply by kerolzeeq** [reply]
Understood. Thanks a lot!

**Reply by chengtan369** [reply]
> They basicly share the samilar architectures. The fast* model use one bert for global and local context, and only forward once. The *s models use syntax-based SRD See ref: https://www.sciencedirect.com/science/article/pii/S0925231220312534 https://aclanthology.org/2020.acl-main.293/

Plus, is *s models way better than standard lcf-atepc?

**Reply by ghost** [reply]
> Plus, is *s models way better than standard lcf-atepc?

I would like to know too!
What do you recommend to use *s models for?

**Reply by yangheng95** [reply]
It is recommanded to use *s models for English language. Generally, *s models are not necessary for large dataset.

**Reply by ghost** [reply]
Where do I find and how do I initialise the *s models? `available_checkpoints(show_ckpts=True)` only shows one checkpoint per language for `APC`:

```
{'multilingual': {'id': '',
  'Training Model': 'FAST-LSA-T-V2-Deberta',
  'Training Dataset': 'APCDatasetList.Multilingual',
  'Language': 'Multilingual',
  'Description': 'Trained on RTX3090',
  'Available Version': '1.10.5+',
  'Checkpoint File': 'fast_lcf_bert_Multilingual_acc_87.18_f1_83.11.zip',
  'Author': 'H, Yang (hy345@exeter.ac.uk)'},
 'multilingual2': {'id': '',
  'Training Model': 'FAST-LSA-T-V2-Deberta',
  'Training Dataset': 'APCDatasetList.Multilingual',
  'Language': 'Multilingual',
  'Description': 'Trained on RTX3090',
  'Available Version': '1.10.5+',
  'Checkpoint File': 'fast_lcf_bert_Multilingual_acc_82.66_f1_82.06.zip',
  'Author': 'H, Yang (hy345@exeter.ac.uk)'},
 'english': {'id': '',
  'Training Model': 'FAST-LSA-T-V2-Deberta',
  'Training Dataset': 'APCDatasetList.English',
  'Language': 'English',
  'Description': 'Trained on RTX3090',
  'Available Version': '1.10.5+',
  'Checkpoint File': 'fast_lsa_t_v2_English_acc_82.21_f1_81.81.zip',
  'Author': 'H, Yang (hy345@exeter.ac.uk)'},
 'chinese': {'id': '',
  'Training Model': 'FAST-LSA-T-V2-Deberta',
  'Training Dataset': 'APCDatasetList.Chinese',
  'Language': 'Chinese',
  'Description': 'Trained on RTX3090',
  'Available Version': '1.10.5+',
  'Checkpoint File': 'fast_lsa_t_v2_Chinese_acc_96.0_f1_95.1.zip',
  'Author': 'H, Yang (hy345@exeter.ac.uk)'}}
```
